custom_components/opnsense/coordinator.py

```
@@ -70,12 +92,240 @@ async def _get_states(self, categories: list) -> MutableMapping[str, Any]:
 
         return state
 
+    def _build_categories(self) -> list[MutableMapping[str, str]]:
```

Pylint (3.3.7)

[refactor] Too many branches (13/12)

(R0912)